### ISSUE TITLE
Feature Request: Export Collision Shapes

### ISSUE THREAD

**Issue by SaracenOne** [godot-extended-libraries/gltf#1]
Would it be possible to have an option export collision shapes as meshes during the export process by triangulating them into meshes?

**Comment by fire** [comment]
Yeah. It's possible. Hm and we can use that special syntax that godot uses.
